packages/backend/src/plugins/catalog.ts

```diff
@@ -15,12 +15,28 @@
  */
 
 import {
-  DatabaseCatalog,
+  DatabaseItemsCatalog,
+  DatabaseLocationsCatalog,
+  DatabaseManager,
   createRouter,
+  runPeriodically,
+  LocationReaders,
+  DescriptorParsers,
 } from '@backstage/plugin-catalog-backend';
 import { PluginEnvironment } from '../types';
 
-export default async function({ logger, database }: PluginEnvironment) {
-  const catalog = await DatabaseCatalog.create(database, logger);
-  return await createRouter({ catalog, logger });
+export default async function ({ logger, database }: PluginEnvironment) {
+  const reader = LocationReaders.create();
+  const parser = DescriptorParsers.create();
```

Comment:
at this point, the user is free to change to instead use

- their own readers - if they have internal systems that need auth so they want to make their own reader, or if they have data stored on other types of system than we have support for 
- their own parsers - if they have files on a different descriptor format than our own